lib/routes/account.js

```diff
@@ -37,6 +37,8 @@ module.exports = function (
     )
   ]
 
+  var verificationReminder = require('../verification-reminders')(log, db)
```

Comment:
Eh, what are your motivations? I'm not convinced we need to, I think push was moved there for some testing reasons?